composer.json

```diff
@@ -74,6 +74,13 @@
             },
             "drupal/redirect": {
                 "Provided config does not match the config after an export": "https://www.drupal.org/files/issues/2932248-3.patch"
+            },
+            "drupal/scheduler": {
+                "Scheduler integration with core Content moderation": "https://www.drupal.org/files/issues/2018-05-07/scheduler-content-moderation-integration-2798689-63.patch"
+            },
+            "drupal/core": {
+                "Move content_moderation config in standard profile": "https://www.drupal.org/files/issues/2018-03-13/move-content_moderation-config.patch",
+                "UpdatePathTestBase fails to pick up new modules": "https://www.drupal.org/files/issues/2018-05-24/2975081-8.5.3-2.patch"
```

Comment:
This patch does not seem to be relevant for users. Patches that are only relevant for our tests should go into the apply_patches function in https://github.com/BurdaMagazinOrg/thunder-distribution/blob/8.x-2.x/scripts/travis/04-install-thunder.sh 
This should also take care of the failing drush make tests.